### PR TITLE
[#293] container: Add IterateContainerSizes method

### PR DESCRIPTION
**Pull request by vdomnich-yadro**
Add method that allows to iterate over estimation records.

Update tests to assert that list of estimations built with existing methods is identical to estimations from iterator.

### CODE REVIEW

File: container/container_contract.go [hunk 1]
```diff
@@ -582,6 +582,19 @@ func ListContainerSizes(epoch int) [][]byte {
 	return result
 }
 
+// IterateContainerSizes method returns iterator over container size estimations
+// that have been registered for the specified epoch.
+func IterateContainerSizes(epoch int) iterator.Iterator {
```
Comment (fyrchik) [inline container/container_contract.go:587]: Do we lose the information about the container in the result iterator? It seems just a key+size pair.

File: container/container_contract.go [hunk 2]
```diff
@@ -551,7 +551,7 @@ func GetContainerSize(id []byte) containerSizes {
 }
 
 // ListContainerSizes method returns the IDs of container size estimations
-// that has been registered for the specified epoch.
+// that have been registered for the specified epoch.
```
Comment (fyrchik) [inline container/container_contract.go:554]: Seems unrelated to this commit.

File: container/container_contract.go (hunk 1 above)
Comment (fyrchik) [reply container/container_contract.go:587]: Key is the key of a storage node, I doubt we can restore container ID from it.

Comment (fyrchik) [reply container/container_contract.go:587]: I think I don't understand the use-case. Will it be sufficient to just return total size for the epoch?
If not, what is done with the values obtained via this method?

Comment (alexvanin) [reply container/container_contract.go:587]: I think we should return both key and value, so container client will define iteration algorithm according to its needs.
As for total size, I agree that separate method will be sufficient.

**Review by fyrchik — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]
@fyrchik Are we going to merge it in support branch or master? Right now target is support/v0.16
